LunaMary/src/Main.java

```diff
@@ -1,14 +1,155 @@
+        quizList.add(quiz1);
+        quizList.add(quiz2);
+
+        List<AnswerPerson> answerPeople = new ArrayList<AnswerPerson>();
+
+        QuizLibrary quizLibrary1 = new QuizLibrary(quizList, answerPeople, "Psicologico");
+        QuizLibrary quizLibrary2 = new QuizLibrary(quizList, answerPeople, "Matematico");
+
+        List<QuizLibrary> quizLibraryList = new ArrayList<QuizLibrary>();
+        quizLibraryList.add(quizLibrary1);
+        quizLibraryList.add(quizLibrary2);
+
+
+        //Lista para Personas
+        List<Person> personList = new ArrayList<Person>();
+
+        boolean exit = false;
+        print("Welcome to your Quiz");
+
+        while (!exit) {
 
+            print("Please enter an option");
+            print("1. Register yourself");
+            print("2. Exit");
+            String option = scanner.nextLine();
+            switch (option) {
+                case "1":
+                    registerPerson(personList);
+                    print("The size Person is: " + personList.size());
+                    break;
+                default:
+                    exit = true;
+            }
 
+
+            while (!exit) {
+                print("Please select a new Quiz");
+                print("1. Psicologico");
+                print("2. Matematico");
+                print("2. Exit");
+                String optionQuiz = scanner.nextLine();
+                switch (optionQuiz) {
+                    case "1":
+                        showQuiz("Psicologico", quizLibraryList);
+                        break;
+                    case "2":
+                        showQuiz("Psicologico", quizLibraryList);
+                        break;
+                    default:
+                        exit = true;
+                }
+
+
+            }
+        }
+
+
+    }
+
+
+    private static void registerPerson(List<Person> personList) {
+        print("Ingresa tu nombre? ");
+        String name = scanner.nextLine();
+        print("Ingresa tu edad? ");
+        String age = scanner.nextLine();
+        print("Ingresa tu sexo  Ejemplo: F  o  M ? ");
+        String sex = scanner.nextLine();
+
+        Person person = new Person();
+        person.setNamePerson(name);
+        person.setAgePerson(age);
+        person.setSexPerson(sex);
+        personList.add(person);
+
+    }
+
+    private static void showQuiz(String type, List<QuizLibrary> quizLibraryList) {
+
+        List<Quiz> quizListRes;
+        Quiz quiz;
+        for (QuizLibrary p : quizLibraryList) {
+
+            if (type.equals(p.getType())) {
+                quizListRes = p.getQuizList();
+                quiz = selectRandomQuiz(quizListRes);
+                List<IQuestion> lisIQuiz = quiz.getiQuestions();
+                for (IQuestion i : lisIQuiz) {
+                    print(i.display());
+                    String optionAnswer = scanner.nextLine();
+                    i.saveAnswer(optionAnswer);
+                }
+            }
```

Comment:
If there is not a single Quiz of the requested type something should be printed on the screen. To explain what happened